spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -452,8 +454,6 @@ def merge_dict(dict_in):
           (1): {'Level': 1, 'Mean(area): 0.2, 'Mean(angle_RL): 12}
       }
 
-    :param dict_in:
-    :return:
```

Comment:
[New](https://spinalcordtoolbox.com/en/aj-2951-fix_doc_warnings/dev_section/api.html#spinalcordtoolbox.aggregate_slicewise.merge_dict), [Old](https://spinalcordtoolbox.com/en/latest/dev_section/api.html#spinalcordtoolbox.aggregate_slicewise.merge_dict)

Do we not want to keep these, even though they're blank? i.e. As a reminder to fill in descriptions once we decide to improve documentation for the API.

There seem to be a lot of blank parameter descriptions across the API, now that I look at it... :stuck_out_tongue:

Reply:
> As a reminder to fill in descriptions once we decide to improve documentation for the API.

yes, that was the idea. It's usually my fault in 99% of the "blank" cases. My IDE automatize those placeholders, and i'm just leaving them as "todo".